app/org/sagebionetworks/bridge/models/ClientInfo.java

```diff
@@ -134,6 +152,13 @@ public Integer getSdkVersion() {
         return sdkVersion;
     }
     
+    public boolean isSupportedVersion(Integer minSupportedVersion) {
+    	if (appVersion != null && minSupportedVersion != null && appVersion < minSupportedVersion) {
+    		return false;
+    	}
+    	return true;
+    }
+    
```

Comment:
yeah, I was trying to follow the style of the other method. Figured that the explicit "this is false and this is true" was preferred.